tests/test_accounts.py

```diff
@@ -228,7 +229,8 @@ def test_get_bulk_by_account_ids(self, flask_test_client):
                     "azure_ad_subject_id": response.json["azure_ad_subject_id"],
                     "email_address": response.json["email_address"],
                     "full_name": None,
-                    "roles": []
+                    "roles": [],
+                    "highest_role": get_highest_role([])
```

Comment:
Is this not cheating, using the response data as the expected data ? 🤔

Reply:
you're right! have changed this now so I am not cheating anymore 😢